docs/_index.md

```diff
@@ -18,7 +18,7 @@ Cortex provides horizontally scalable, highly available, multi-tenant, long term
 [Prometheus](https://prometheus.io).
 
 - **Horizontally scalable:** Cortex can run across multiple machines in a cluster, exceeding the throughput and storage of a single machine. This enables you to send the metrics from multiple Prometheus servers to a single Cortex cluster and run "globally aggregated" queries across all data in a single place.
-- **Highly available:** When run in a cluster, Cortex can replicate data between machines. This allows you to survive machine failure without gaps in your graphs.
+- **Highly durability:** When run in a cluster, Cortex can replicate data between machines. This allows you to survive machine failure without gaps in your graphs.
```

Comment:
This change is incorrect. Highly available is correct on this case, because it refers to the in-memory series in the ingesters before they're flushed to the long-term storage. Cortex can tolerate a single ingester node failure.

Reply:
Thank you for your reply. I got it.